readme.md

```diff
@@ -45,6 +46,7 @@ Configure it in `package.json`.
 
 - [catch-error-name](docs/rules/catch-error-name.md) - Enforce a specific parameter name in catch clauses.
 - [filename-case](docs/rules/filename-case.md) - Enforce a case style for filenames.
+- [no-abusive-eslint-disable](docs/rules/no-abusive-eslint-disable.md) - Specify rules to disable when using `eslint-disable` comments
```

Comment:
:+1:, though technically we don't enforce the `exact` rules (as we don't know which ones are useful or not)